scripts/server.js

```
@@ -1,38 +1,61 @@
+
+  async init() {
+    try {
+      const path_userData = app.getPath("userData");
+      const filePath = path.join(path_userData, "server_login.json");
+      const data = JSON.parse(fs.readFileSync(filePath, { encoding: "utf8" }));
+      console.log("server_login: ", data);
+      this.#email = data.email;
+      this.#password = data.password;
+      this.#server_url = data.server_url;
+    } catch (e) {
+      throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)